src/client/renderer/chunk_renderer.cpp

```diff
@@ -80,6 +80,23 @@ namespace {
         }
     }
 
+    /**
+     * @brief Checks whether chunk is within the render distance using L1 distance
+     *
+     * @param cameraPosition The position of the camera (in world coordinates)
+     * @param chunkPosition The position of the chunk (in "chunk" coordinates)
+     * @param renderDistance The render distance
+     */
+    bool isChunkInRenderDistance(const glm::vec3& cameraPosition,
+                                 const ChunkPosition chunkPosition,
```

Comment:
This can be passed by reference 
```suggestion
                                 const ChunkPosition& chunkPosition,
```

Reply:
Oh yes, thanks.